tutor/templates/apps/openedx/settings/lms/production.py

```diff
@@ -49,22 +30,3 @@
 BULK_EMAIL_DEFAULT_FROM_EMAIL = 'no-reply@' + ENV_TOKENS['LMS_BASE']
 API_ACCESS_MANAGER_EMAIL = ENV_TOKENS['CONTACT_EMAIL']
 API_ACCESS_FROM_EMAIL = ENV_TOKENS['CONTACT_EMAIL']
```

Comment:
The bonus thing with this PR is that is highlights the settings that I forgot to copy paste from production to development... I'll take care of moving these settings later.